### ISSUE TITLE
Could you share how many different categories are trained in RL?

### ISSUE THREAD

**Issue by RayYoh** [hyperplane-lab/RLAfford#11]
Hi,

Thanks for your excellent code.
I'm trying to reproduce the results but having some issues like #7 . 
I'm having trouble reproducing the close door results. This is still happening even when I use numAssetTrain=44, numTrain=1 but goes away when I reduce asset count (5) and increase env count (160 or more), so I want to check could you share what asset counts you used in the paper, and my GPU memory is 48GB.

Best

**Comment by boshi-an** [comment]
We used the full training set in our paper. Also, we used a 80G A100 to do the final training, so we didn't encounter any similar issues. It is a fact that the loaded assets takes a lot memory no mater how many times it is instantiated. You may try to lower the numAssetTrain param to 40, 36 or some similar values, not harming the overall performance but still can run on your 48GB GPU.

**Comment by RayYoh** [comment]
> We used the full training set in our paper. Also, we used a 80G A100 to do the final training, so we didn't encounter any similar issues. It is a fact that the loaded assets takes a lot memory no mater how many times it is instantiated. You may try to lower the numAssetTrain param to 40, 36 or some similar values, not harming the overall performance but still can run on your 48GB GPU.

Hello, thanks for your reply.
What I find is that the error occurred even though my GPU memory was not fully occupied.
When I modified the `asset_options` in [franka_cabinet.py](https://github.com/hyperplane-lab/RLAfford/blob/main/MARL_Module/envs/tasks/franka_cabinet.py) like those in [MVP](https://github.com/ir413/mvp/blob/master/pixmc/tasks/franka_cabinet.py), it works although I load all of the train assets and add the num_envs.  So, I'd like to ask whether there are some documentations about those parameters.
```
asset_options = gymapi.AssetOptions()
asset_options.fix_base_link = True
asset_options.disable_gravity = True
asset_options.collapse_fixed_joints = True
asset_options.use_mesh_materials = True
asset_options.mesh_normal_mode = gymapi.COMPUTE_PER_VERTEX
asset_options.override_com = True
asset_options.override_inertia = True
asset_options.vhacd_enabled = True
asset_options.vhacd_params = gymapi.VhacdParams()
asset_options.vhacd_params.resolution = 512
```

**Comment by boshi-an** [comment]
The only document we checked is the official one provided by nvidia, when downloading isaacgym. This error indeed may happen even if GPU memory isn't fully occupied, we observed this phenomenon too. This may due to excess memory used for reservation and memory copy. The modified asset options will limit convex-decomposition precision (asset_options.vhacd_params.resolution = 512), which will lower the memory usage.
